integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/GetWalksIT.java

```diff
@@ -106,7 +107,7 @@ public void shouldGetPaths() throws Exception {
         final Iterable<Walk> results = graph.execute(op, getUser());
 
         // Then
-        assertThat(getPaths(results)).isEqualTo("AED,ABC");
+        assertThat(getPaths(results)).contains("AED", "ABC");
```

Comment:
Would containsExactly be a more suitable replacement?

Reply:
Good spot. This change was done by @t511203 for the Spring update. @t511203 do you agree?